google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -352,6 +356,21 @@ public void writeDateNull() {
     assertThat(row.isNull(0)).isTrue();
   }
 
+  @Test
```

Comment:
@olavloite thanks for adding the integration tests. But I do have one small request. Would you be able to move it to a separate PR?

The reason being that we will merge this PR in before support for the feature is available in prod, which unfortunately mean that the tests will fail at that point.

Reply:
NP, I'll move them to a separate PR.